wasm-rpc-stubgen/src/commands/app.rs

```diff
@@ -394,8 +394,11 @@ async fn gen_rpc<CPE: ComponentPropertiesExtensions>(
         for component_name in ctx.wit.component_order_cloned() {
             create_generated_base_wit(ctx, &component_name)?;
         }
-        for component_name in &ctx.application.all_wasm_rpc_dependencies() {
-            build_stub(ctx, component_name).await?;
+
+        for dep in &ctx.application.all_wasm_rpc_dependencies() {
```

Comment:
maybe we should add a getter named "all_static_wasm_rpc_dependencies" (or we might want to report skipping dynamic ones, which could be useful for debugging, although it would make logs more versbose)

Reply:
👌